.gitignore

```diff
@@ -0,0 +1,32 @@
+# Prerequisites
+*.d
+
+# Compiled Object files
+*.slo
+*.lo
+*.o
+*.obj
+
+# Precompiled Headers
+*.gch
+*.pch
+
+# Compiled Dynamic libraries
+*.so
+*.dylib
+*.dll
+
+# Fortran module files
+*.mod
```

Comment:
Fortran?
Please let's keep only the stuff that actually in use here.
From object files we need only *.o I think. What is *.gch precompiled headers? Who creates them? Do we create any dylib/dll? Do we build any static libraries at all?